doc/source/ray-air/package-ref.rst

```diff
@@ -74,6 +74,15 @@ Feature Scalers
 .. autoclass:: ray.data.preprocessors.StandardScaler
     :show-inheritance:
 
+K-Bins Discretizers
+###################
+
+.. autoclass:: ray.data.preprocessors.CustomKBinsDiscretizer
+    :show-inheritance:
+
+.. autoclass:: ray.data.preprocessors.UniformKBinsDiscretizer
+    :show-inheritance:
```

Comment:
ooc, why'd we create separate classes rather than use a `strategy` like sklearn?

Reply:
This approach seemed cleaner to me - and the custom k-bins discretizer doesn't require fitting, so it makes sense to make it a separate class. The future quantile discretizer will be quite different internally so breaking it up into separate classes will make the code cleaner and easier to understand.